cnf-certification-test/accesscontrol/securitycontextcontainer/securitycontextcontainer.go

```diff
@@ -0,0 +1,103 @@
+package securitycontextcontainer
+
+import (
+	v1 "k8s.io/api/core/v1"
+)
+
+type ContainerSCC struct {
+	HostDirVolumePlugin    bool
+	HostIPC                bool
+	HostNetwork            bool
+	HostPID                bool
+	HostPorts              bool
+	PrivilegeEscalation    bool
+	PrivilegedContainer    bool
+	RunAsUser              bool
+	ReadOnlyRootFilesystem bool
+	RunAsNonRoot           bool
+	Capabilities           string
```

Comment:
Add API references in the code documentation:
https://docs.openshift.com/container-platform/4.11/authentication/managing-security-context-constraints.html
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/
https://andreaskaris.github.io/blog/openshift/scc/
After reading some more about the runasgroup, I think that we should check that it is set to something. It is not specified in the example SCC probably because it is not known in advance, but the mustRunAs seems to suggest that the intent is to have it set (even if the range is not set)